code_examples/sdk_examples/src/core_features/getting_started/04_fetch_endpoints.ts

```diff
@@ -1,18 +1,16 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
+import * as Did from '@kiltprotocol/did'
 
-export async function main(
-  uri: Kilt.DidUri
-): Promise<Kilt.DidServiceEndpoint[]> {
-  const kiltnerd123DidDocument = await Kilt.Did.resolve(uri)
+export async function main(id: String): Promise<Object[]> {
+  const kiltnerd123DidDocument = await Did.resolve(id)
```

Comment:
`Kilt.DidResolver.resolve()` I think?

Reply:
Its fixed in: https://github.com/KILTprotocol/docs/pull/329/commits/2fb4bf229be1bd8c28ed45066790a66d8b5f1e29